life_dashboard/life_stats/views.py

```
@@ -1,3 +1,4 @@
 from django.shortcuts import render
+from life_dashboard.life_stats.models import LifeStat, LifeStatCategory
```

Ruff (0.11.9)

`life_dashboard.life_stats.models.LifeStat` imported but unused

Remove unused import

(F401)